staging/cse/windows/azurecnifunc.ps1

```diff
@@ -67,8 +69,7 @@ function Set-AzureCNIConfig
         if ($IsDualStackEnabled){
             $subnetToPass = $KubeClusterCIDR -split ","
             $exceptionAddresses = @($subnetToPass[0])
-        }
-        else {
+        } elseif (!$IsAzureCNIOverlayEnabled) {

```

Comment:
Does `elseif` mean that AzureCNIOverlay cannot work with dualstack?

Reply:
these are auto gofmt changes

Reply:
I'm not sure about this, let me discuss w/ team

Reply:
We do not support dual stack for now but we plan to support ipv6 soon. Do you have a plan to support overlay with ipv6?